### ISSUE TITLE
Serilog doesn't log values after the switch to Microsoft.Extensions.Logging.Abstractions

### ISSUE THREAD

**Issue by eriove** [helix-toolkit/helix-toolkit#1811]
After trying latest from the develop branch I noticed that the logs suddenly contained `{}` instead of values. This is due to he switch to Microsoft.Extensions.Logging.Abstractions. We use Serilog in the background which uses its own formatter instead of the one provided by Microsoft.Extensions.Logging. This formatter treats {} as a string instead of a location to insert values.

I've made a [pull request in serilog-extensions-logging](https://github.com/serilog/serilog-extensions-logging/pull/206) instead of here since this seems like a general problem with using the combination of Serilog and Microsoft.Extensions.Logging that more people could benefit from getting fixed.

I've just created this issue in case someone else encounters the same problem.

**Comment by holance** [comment]
https://github.com/serilog/serilog-extensions-logging
I think you can switch to this easily without too many changes?

**Comment by eriove** [comment]
Yes, switching is two lines of code. But after switching the {} in the logging templates isn't replaced by actual values so the logs aren't as useful. That's an oversight in serilog-extensions-logging and not something that should be fixed here, I just created the issue for others that might run into the same problem. Once it is fixed in serilog-extensions-logging it isn't needed any more.

As mentioned I've made a pull request in https://github.com/serilog/serilog-extensions-logging. Hopefully the maintainers of that library will merge the pull request and then it will work just as before.
